src/schemas/execute.yaml

```diff
@@ -240,7 +240,7 @@ CallResults:
   title: Results of multi call within block
   type: array
   items:
-    oneOf:
+    anyOf:
```

Comment:
I'm pretty sure this should be `oneOf`, not `anyOf`.  A `CallResult` cannot be both a Failure and a Success at the same time.